python/res/fm/ecl/ecl_config.py

```diff
@@ -186,4 +193,49 @@ class FlowConfig(EclConfig):
 
     def __init__(self):
         config_file = os.getenv("FLOW_SITE_CONFIG", default = self.DEFAULT_CONFIG_FILE)
-        super(FlowConfig, self).__init__(config_file)
+        super(FlowConfig, self).__init__(config_file, simulator_name="flow")
+
+
+
+class EclrunConfig:
```

Comment:
The naming here is a bit awkward, as we have the classes `EclConfig` with a subclass (`EclXXXConfig`) for each of the simulators, while `EclrunConfig` is by all means not a subclass of `EclConfig`, and instead is ment to be used in addition to `EclConfig`. I do understand the issue of having a configclass for a binary named `eclrun`, and then what do you name it.. But at least this was a bit confusing

Reply:
Completely agree, but I was not able to come up with any better naming! Very open to suggestions

Reply:
I hate how annoying naming can be, and in this case specifically, where we end up in a slight pinch due to naming of the binary which is completely out of our hands.

Please provide some documentation to the `EclrunConfig`, as to what it is (and highlight the difference in relation to the other `Ecl*Config` classes), and then I think we are good to go.